spec/integration/_dev/deploy/tf/files/spec.yml

```
@@ -0,0 +1,18 @@
+spec:
+  additionalContents: false
+  contents:
+  - description: Log / conf files to add into terraform resources
+    type: file
+    contentMediaType: "text/plain"
+    pattern: '^*.(log|conf)$'
+    required: false
+  - description: YAML files to add into terraform resources
+    type: file
+    contentMediaType: "application/yml"
```

Sorry, I didn't see it before. I think it should be `application/x-yaml`.
```suggestion
    contentMediaType: "application/x-yaml"
```